tests/unit/views/notifications/project_crew_notification.feature

```diff
@@ -145,3 +145,23 @@ Feature: Project Crew Notification
       | Ridcully | usher           | You resigned from the crew of Ankh-Morpork 2010                       |
       | Vetinari | usher           | Mustrum Ridcully resigned from the crew of Ankh-Morpork 2010          |
       | Vimes    | usher           | Mustrum Ridcully resigned from the crew of Ankh-Morpork 2010          |
+
+  Scenario Outline: Ridcully adds themself
+    Given Vetinari made Ridcully an admin of Ankh-Morpork
+    When Ridcully adds themself with role <role> to the Ankh-Morpork 2010 project
+    Then <user> gets notified with <notification_string> about the addition
+
+    Examples:
+      | user     | role            | notification_string                                                         |
+      | Ridcully | editor          | You added yourself as editor of Ankh-Morpork 2010                           |
+      | Vetinari | editor          | Mustrum Ridcully added themself as editor of Ankh-Morpork 2010              |
```

Comment:
"Themself" is a relatively new gender-neutral reference in English. It won't work in Hindi and most other languages that have not yet tackled gender-neutral pronouns. We can make it easier for translators by saying it like this (although it's a bit ambiguous about who the actor is):

"Mustrum Ridcully joined Ankh-Morpork 2010 as editor"
"Mustrum Ridcully joined the crew of Ankh-Morpork 2010"